### DISCUSSION TITLE
What's missing from the site and the book?

### DISCUSSION THREAD

**Discussion by andrewharmellaw** [andrewharmellaw/facilitating-software-architecture#2] [Q&A]
Hi there,
My biggest aim for the book, [this site](https://facilitatingsoftwarearchitecture.com), and the community as a whole is that it _helps people deliver better software_. I've tried to put the most relevant information in the book, and other "important but not essential" (thanks Rita) information here. 

But both these acts are kind of guesses. I'm hoping I got it right, but its your feedback that will tell me if I managed it. 

If there is something you are looking for, but that you can't find, please add it here, and I'll be able to either fix it (or someone else will before I get there), or I'll disappoint you quickly by suggesting an alternative.

**Comment by mduesterhoeft** [comment]
One thing that is still a bit blurry is the way from decided ADR to implementation. So something is agreed but the work is not prioritized because other things are considered more important. Should one take the likelihood of priotization of work already in to account when deciding which decision to propose?

**Reply by andrewharmellaw** [reply]
Great question @mduesterhoeft. Thank you. I'd hope that this could be brought into the context of a decision, and also pros and cons for various options. I've frequently seen Product Managers offer exactly this kind advice. They are the people (or POs in other dev approaches) who ought to know what the team's capacity is capable of. Decisions that don't make it into the code are sadly frequently a waste of time.

**Reply by andrewharmellaw** [reply]
(also makes a note to put this in a future version of the book. 📝 )

**Reply by mduesterhoeft** [reply]
Thanks a lot - My own immediate reaction was that we should not look at priotization and capacity too much in an ADR. So we can focus on e.g. the technical direction we want to take. But you are right - if the ADR is never priotized this is just waist (and confusing for readers of the ADR)

**Reply by andrewharmellaw** [reply]
The key thing to bear in mind is that a decision is "“a conscious choice between two or more alternatives that involves an irrevocable allocation of resources”. (From "[Collaborative Software Design](https://www.goodreads.com/book/show/199324780-collaborative-software-design)" by Gien Verschaste, Evelyn van Kelle and Kenny Baas-Schwegler). Its not always a key factor in decisions, but it can help a lot to keep it in mind - it can lead to better, smaller, more focused and appropriate decisions.
